### PR TITLE
Fix doctrine deprecations #1

### PR DESCRIPTION
**Pull request by XWB**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "require-dev": {
         "symfony/phpunit-bridge": "^2.8|^3.0|^4.0|^5.0",
         "doctrine/common": "~2.2",
+        "doctrine/persistence": "^1.3.3",
```
Comment (nicolas-grekas) [inline composer.json]: if we do this, php should be bumped to >=7.1 above

Comment (XWB) [reply composer.json]: That means Symfony 2.8 support should be dropped as well. Is that something we can do?

Alternatively I can hack something together, like https://github.com/symfony/doctrine-bridge/commit/e3671a2fa2ed4339cea11b6e889639b1e986542e

Comment (jordisala1991) [reply composer.json:25]: is bumping php to 7.1 acceptable here or do we have to find alternative solution? cc @fabpot

**Comment by fabpot** [comment]
Thank you @XWB.

**Pull request merged**